scripts/validate_docstrings.py

```diff
@@ -838,8 +842,16 @@ def header(title, width=80, char='#'):
                            'list of error codes to validate. By default it '
                            'validates all errors (ignored when validating '
                            'a single docstring)')
+    argparser.add_argument('--deprecated', default=False, type=bool,
```

Comment:
The argument name should probably be `--ignore-deprecated`, and just deprecated is not very clear